test/service/test_response.py

```diff
@@ -1420,6 +1420,27 @@ def test_ordering(self):
             self.assertEqual(hit_sort_values['asc'], sorted(hit_sort_values['asc']))
             self.assertEqual(hit_sort_values['desc'], sorted(hit_sort_values['desc'], reverse=True))
 
+    def test_multivalued_field_sorting(self):
+        """
+        Test that sorting by a multi-valued field responds with hits that are
+        correctly sorted based on the first value from each multi-valued field, and
+        that each multi-valued field itself is sorted low to high regardless of the search sort
```

Comment:
Is the statement in the last line true?

```suggestion
        that each multi-valued field itself is sorted low to high regardless of the search sort.
```

Reply:
Yes. This test uses the `laboratory` field to verify 2 aspects of sorting:
- A request made using `sort` and `order` parameters against a multi-valued field like `laboratory` will order the hits based on the first (smallest) element from each hit's `laboratory` field. 
```
1442                self.assertEqual(laboratories, sorted(laboratories, reverse=reverse))
```
- Multi-valued fields (only `laboratory` verified here) is itself sorted low to high
```
1439                    self.assertEqual(laboratory, sorted(laboratory))
```

Reply:
I see, thank you!